packages/date-picker/test/dropdown.common.js

```diff
@@ -282,6 +282,20 @@ describe('dropdown', () => {
 
       expect(datePicker.hasAttribute('focus-ring')).to.be.true;
     });
+
+    it('should stop click events from bubbling outside the overlay', async () => {
```

Comment:
Is it possible to test the use-case in which the bug occurs = something like "should not close another overlay..." ?

Testing that a click event shouldn't bubble is testing an implementation detail.

Reply:
I don't have a strong opinion, it should be possible but I couldn't think of a more generalized case. I am wondering whether it would also be testing the behaviour of the other overlay in that case.

Reply:
> I am wondering whether it would also be testing the behaviour of the other overlay in that case.

The test case could be added to [`integration` tests](https://github.com/vaadin/web-components/tree/main/integration/tests). Would be nice if the test was still valid in case the fix is later changed to something else.

Reply:
OK, this makes perfect sense. I will update the test.

Reply:
Updated the tests.